server/repository/src/db_diesel/assets/asset_catalogue_item.rs

```diff
@@ -154,13 +154,25 @@ fn create_filtered_query(filter: Option<AssetCatalogueItemFilter>) -> BoxedAsset
             code,
             manufacturer,
             model,
-            ..
+            category: _, // Handled in query() function
```

Comment:
FYI - @mark-prins I updated this as it looks like you'd missed the new id filters.
This pattern of marking something as unused will warn you for anything new added so it won't be missed.